File: app/src/main/java/com/footinit/motionlayoutplayground/MyApp.java
package com.footinit.motionlayoutplayground;

import android.app.Application;
import android.content.Context;

import androidx.multidex.MultiDex;
import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
import uk.co.chrisjenx.calligraphy.CalligraphyConfig.Builder;

public class MyApp extends Application {

    public void onCreate() {
        super.onCreate();
        CalligraphyConfig.initDefault(
                new Builder()
                        .setDefaultFontPath("fonts/Google-Sans-Font/GoogleSans-Regular.ttf")
                        .setFontAttrId(R.attr.fontPath).build());
    }

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: app/src/main/java/com/footinit/motionlayoutplayground/MyApp.kt
package com.footinit.motionlayoutplayground

import android.app.Application
import com.footinit.motionlayoutplayground.di.appModules
import org.koin.android.ext.koin.androidContext
import org.koin.android.ext.koin.androidLogger
import org.koin.core.context.startKoin

class MyApp : Application() {

    override fun onCreate() {
        super.onCreate()

        startKoin {
            androidContext(this@MyApp)
            androidLogger()
            modules(appModules)
        }
    }
}